### ISSUE TITLE
Training and Validation Accuracy

### ISSUE THREAD

**Issue by tengen119** [Maxhin/PyTorch_Malaria_Detection#2]
![Malaria_Tr_val_acc](https://user-images.githubusercontent.com/60868520/95409426-8781bd00-095c-11eb-999c-03dc611b55bf.png)
